suites/pacific/rbd/tier-2_rbd_mirror_regression.yaml

```diff
@@ -161,82 +163,88 @@ tests:
 
-  - test:
-      name: Recovery of abrupt failure of primary cluster
-      module: test_9470.py
-      clusters:
-        ceph-rbd1:
-          config:
-            imagesize: 2G
-            io-total: 200M
-      polarion-id: CEPH-9470
-      desc: Test unplanned failover and failback scenario
+  # - test:
+  #     name: test image meta operations sync to secondary
+  #     module: test_rbd_image_meta_mirroring.py
+  #     clusters:
+  #       ceph-rbd1:
+  #         config:
+  #           imagesize: 2G
+  #           io-total: 200M
+  #           key: ping
+  #           value: pong
+  #     polarion-id: CEPH-9524
+  #     desc: Verify removal of image meta gets mirrored
 
-  - test:
-      name: Recovery of shutdown primary cluster
-      module: test_9471.py
-      clusters:
-        ceph-rbd1:
-          config:
-            imagesize: 2G
-            io-total: 200M
-      polarion-id: CEPH-9471
-      desc: Test planned failover and failback scenario
+  # - test:
+  #     name: Recovery of abrupt failure of primary cluster
+  #     module: test_9470.py
+  #     clusters:
+  #       ceph-rbd1:
+  #         config:
+  #           imagesize: 2G
+  #           io-total: 200M
+  #     polarion-id: CEPH-9470
+  #     desc: Test unplanned failover and failback scenario
 
-  - test:
-      name: Recovery of shutdown secondary cluster
-      module: test_9475.py
-      clusters:
-        ceph-rbd1:
-          config:
-            imagesize: 2G
-            io-total: 200M
-      polarion-id: CEPH-9475
-      desc: Testing secondary cluster unplanned failover test
+  # - test:
+  #     name: Recovery of shutdown primary cluster
+  #     module: test_9471.py
+  #     clusters:
+  #       ceph-rbd1:
+  #         config:
+  #           imagesize: 2G
+  #           io-total: 200M
+  #     polarion-id: CEPH-9471
+  #     desc: Test planned failover and failback scenario
 
-  - test:
-      name: Mirroring of cloned image
-      module: test_rbd_clone_mirror.py
-      polarion-id: CEPH-9521
-      desc: Testing mirroring of cloned image
+  # - test:
+  #     name: Recovery of shutdown secondary cluster
+  #     module: test_9475.py
+  #     clusters:
+  #       ceph-rbd1:
+  #         config:
+  #           imagesize: 2G
+  #           io-total: 200M
+  #     polarion-id: CEPH-9475
+  #     desc: Testing secondary cluster unplanned failover test
 
-  - test:
-      name: Mirroring from journal to snapshot
-      module: test_rbd_mirror_journal_to_snap.py
-      polarion-id: CEPH-83573618
-      desc: Testing journal mirroring to snapshot mirroring
+  # - test:
+  #     name: Mirroring of cloned image
+  #     module: test_rbd_clone_mirror.py
+  #     polarion-id: CEPH-9521
+  #     desc: Testing mirroring of cloned image
+
+  # - test:
+  #     name: Mirroring from journal to snapshot
+  #     module: test_rbd_mirror_journal_to_snap.py
+  #     polarion-id: CEPH-83573618
+  #     desc: Testing journal mirroring to snapshot mirroring
```

Comment:
These commenting out needs to be reverted right ?
@rkhambad

Reply:
yeah those will be reverted

Reply:
Reverted.